lib/plaidRequest.js

```diff
@@ -71,6 +71,10 @@ var plaidRequest = function(context, requestSpec, clientRequestOptions, cb) {
     headers['Plaid-Version'] = clientRequestOptions.version;
   }
 
+  if (clientRequestOptions.clientApp != null) {
+    headers['Client-App'] = clientRequestOptions.clientApp
```

Comment:
should this be something like `Plaid-Client-App` ?

Reply:
:zap: agreed and was my initial choice